.gitignore

```diff
@@ -4,7 +4,8 @@
 *.css
 /src/*.txt
 /src/*.oct
-*.mat
```

Comment:
instead of changing this i suggest not doing any change in the gitignore and if u want to add a .mat file you can force it with `git add -f file.mat` or something like that

Reply:
Done